### ISSUE TITLE
Automated Database Seed for Cypress Testing 

### ISSUE THREAD

**Issue by abschwenker** [bcgov/SIMS#1494]
Additional Context
- In order to support Cypress, additional dev work is required to seed the database for required data.
- The final goal is to have a QA DB seeding solution that can be executed in the QA environment directly or in the future in a docker environment.

**Acceptance Criteria**
- [x] Do research to look for possible best practices on Cypress guidelines or in any other approach used to seed the database;
- [x] Create a solution to execute the DB scripts needed for seeding the data in a test case isolated way. For instance, for a particular test case, the code structure should be easy to understand and must consolidate all the DB changes needed to execute the particular test case.
- [x] Implement one data seed using one of the test scenarios already in place, for instance, a designation agreement.
- [ ] Explore how we can run altogether from docker-compose, similar to what is happening in `local-api-test` that is also running on GitHub actions.

**Initial thoughts**
- Create a new backend app and consume the @sims-db library;
- Create service-like classes to execute the DB commands similar to the other services in API/Workers;
- Execute all the service methods to seed the DB in a parallel way.

**Comment by ann-aot** [comment]
As part of my research, I found the below links
https://learn.cypress.io/advanced-cypress-concepts/database-initialization-and-seeding
which basically list almost all the approach we discussed.
`Custom scripts (factories) to generate "test" data` are something similar that we are planning to do as part of this ticket with Faker.js.

Below are the methods in cypress with which we execute or run our scrips
https://docs.cypress.io/faq/questions/using-cypress-faq#How-do-I-seed-reset-my-database
Below cypress function will help to execute a system command. 
![image.png](https://images.zenhubusercontent.com/608872f7d0d25cdfe8dab75d/214cca8d-ff8e-4e9e-87a9-c6e6f7c563fe)

Below cypress function will help Execute code in Node via the task plugin event.
![image.png](https://images.zenhubusercontent.com/608872f7d0d25cdfe8dab75d/3da80812-bf7e-499b-9dbd-de8fa1539d2a)

As part of this story we are planning to run / exceute the script via Github actions.
